docs/onboard.md

```diff
@@ -0,0 +1,86 @@
+# Onboard
+
+## Vendor
+
+### Step 1
+Send in your request for a TechPass account to TechPass Team
```

Comment:
For vendor accounts, they need to be invited by tenant admins so that needs to be stated here.